services/movie.py

```diff
@@ -0,0 +1,53 @@
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+    genres_ids: list[int] = None,
+    actors_ids: list[int] = None
+) -> QuerySet:
+    movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        movies = movies.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        ).distinct()
+    elif genres_ids:
+        movies = movies.filter(genres__id__in=genres_ids).distinct()
+    elif actors_ids:
+        movies = movies.filter(actors__id__in=actors_ids).distinct()
```

Comment:
```suggestion
    if genres_ids:
        movies = movies.filter(genres__id__in=genres_ids).distinct()
    if actors_ids:
        movies = movies.filter(actors__id__in=actors_ids).distinct()
```
Make your code simple as possible